docker-compose-swarm.yml

```diff
@@ -11,15 +12,19 @@ consul:
     - 8500:8500
     restart: always
 
+# Run the application on AWS/GCE as follows:
+# sed -r "s/consul:<CONSUL_IP>/consul:$(shell cat .aws/consul)/" ../docker-compose-swarm.yml > /tmp/swarm.yml
+# COMPOSE=/tmp/swarm.yml PREFIX=cb4 VERSION=4 ../start.bash
 
 # Manually bootstrap the first instance, then...
 # Scale this tier and each additional container/instance will automatically
 # self-configure as a member of the cluster
 couchbase4:
     image: 0x74696d/triton-couchbase:4.0.0
+    net: host
     extra_hosts:
-    - consul:${CONSUL_IP}
-    mem_limit: 128g
+    - "consul:<CONSUL_IP>"
```

Comment:
It's just to mark this for the line of sed here: https://github.com/misterbisson/clustered-couchbase-in-containers/pull/10/files#diff-edbb3d293f4a3df14cbfde338382c9dcR132

You can't pass environment variables from the client's host into docker-compose, so I'm injecting them via sed and then using that temp file as the input to docker-compose.